src/app/login-signup/sign-up/sign-up.component.html

```diff
@@ -22,28 +69,59 @@
 					[(ngModel)]="password"
 					name="password"
 					spellcheck="false"
+					(focus)="showTooltipP = true"
+					(blur)="showTooltipP = false"
 				>
 
+				<app-icon
+					*ngIf="validateValuesService.validatePassword(password).correct"
+					class="icon x-check"
+					iconName="check"
+					iconSize="20"
+				></app-icon>
+
+				<app-icon
+					*ngIf="!validateValuesService.validatePassword(password).correct"
+					class="icon x-check"
+					iconName="times"
+					iconSize="15"
+				></app-icon>
+
 				<app-icon
 					class="icon"
 					iconName="eye"
 					iconSize="25"
 					isButton="true"
-					(click)="passwordInput.type = 'password' "
-					*ngIf="passwordInput.type == 'text' "
+					(click)="passwordInput.type = 'password'"
+					*ngIf="passwordInput.type == 'text'"
 				></app-icon>
 
 				<app-icon
 					class="icon"
 					iconName="eye-slash"
 					iconSize="25"
 					isButton="true"
-					(click)="passwordInput.type = 'text' "
-					*ngIf="passwordInput.type == 'password' "
+					(click)="passwordInput.type = 'text'"
+					*ngIf="passwordInput.type == 'password'"
 				></app-icon>
 			</div>
+			<!-- Password End -->
 
+
+			<!-- Confirm Password Start -->
 			<div class="input-container">
+
+				<div
+					class="tooltip-container"
+					*ngIf="showTooltipCP"
+				>
+					<span class="tooltip bad"
+						[class.good]="(password === confirmPassword && password.length !== 0)"
+					>
+						{{ password === confirmPassword && password.length !== 0 ? "Looks good!" : "Passwords must match!" }}
```

Comment:
You might want to add some more logic in here... It says "Passwords must match!" when both passwords are "".
```suggestion
						{{ password.length === 0 ? "Your password can't be nothing..." : (password === confirmPassword && password.length !== 0 ? "Looks good!" : "Passwords must match!") }}
```